inductiva/molecules/protein_solvation/README.md

```diff
@@ -31,15 +31,20 @@ Moreover, the hardware and interaction are configured with the usual general par
 
 ### Example
 
-To test this scenario we have available one example of a protein PDB file - [download it here](inductiva/resources/alanine.pdb).
+To test this scenario you can download a protein PDB file from the RCSB database using the Inductiva package.. Below, we download the insulin protein with the PDB ID "1ZNI".
+
+Do not forget to insert your API Key (get one by filling this [form](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform?usp=sf_link)).
+
 
 ```python
- import inductiva
+import inductiva
 
 inductiva.api_key = "YOUR_API_KEY"
 
+# Download the insulin protein (ID - "1ZNI") from RCSB database
+insulin = inductiva.molecules.utils.download_pdb_from_rcsb(pdb_id="1ZNI")
```

Comment:
insulin_pdb

(otherwise it looks like this creates some python object of a protein, but it's just a string with a local path)